app/src/test/java/com/kickstarter/viewmodels/NativeCheckoutRewardViewHolderViewModelTest.kt

```diff
@@ -112,6 +117,33 @@ class NativeCheckoutRewardViewHolderViewModelTest : KSRobolectricTestCase() {
         this.backersCountIsGone.assertValue(true)
     }
 
+    @Test
+    fun testBackground_whenReward() {
+        setUpEnvironment(environmentWithGoRewardlessDisabled())
```

Comment:
Did you mean for this to be enabled, so you can test that a regular reward with the `Go Rewardless` feature _enabled_ doesn't apply a background? Or maybe that needs a different test?

Reply:
hello and thank you for looking! yeah, i think i'll need another test! should i also add one for `testDescriptionOutputs_whenReward_goRewardlessEnabled`. There a lot of tests 😓

Reply:
Added `testBackground_whenReward_goRewardlessEnabled` and renamed this one to `testBackground_whenReward_goRewardlessDisabled`